src/model/Environment.ts

```diff
@@ -45,17 +44,29 @@ export enum Status {
 };
 
 interface DeploymentState {
-  crons: object;
+  crons: {
+    enabled: boolean;
+    status: "running" | "sleeping";
```

Comment:
Looks great. Only think I see is that the status "sleeping" has been changed to "paused"

Reply:
Thanks for catching that!